packages/ui/src/theme/css/styles.scss

```diff
@@ -3,6 +3,7 @@
  */
 
 @import 'dist/theme';
+@import './component/fieldControl';
```

Comment:
Moving this to line 54? That way we can have all component styles group together

Reply:
yes that has been moved